regression-test/suites/query_p0/sql_functions/datetime_functions/test_date_floor_ceil.groovy

```diff
@@ -32,5 +32,20 @@ suite("test_date_floor_ceil") {
     qt_sql9 """select date_ceil("2023-07-14 10:51:00",interval 5 hour); """
     qt_sql10 """select date_ceil("2023-07-14 10:51:00",interval 5 day);   """
     qt_sql11 """select date_ceil("2023-07-14 10:51:00",interval 5 month); """
-    qt_sql12 """select date_ceil("2023-07-14 10:51:00",interval 5 year); """    
+    qt_sql12 """select date_ceil("2023-07-14 10:51:00",interval 5 year); """
+
+    qt_x1 """ select date_floor('9999-12-31 23:59:59.999999', interval 5 minute); """
+    qt_x2 """ select date_floor('9999-12-31 23:59:59.999999', interval 33333 year); """
+    qt_x3 """ select date_floor('9999-12-31 23:59:59.999999', interval -10 year); """
+    qt_x4 """ select date_floor('1923-12-31 23:59:59.999999', interval -10 year); """
+    // qt_x5 """ select date_floor('0000-01-01 00:00:00', interval 7 minute); """//wrong
```

Comment:
what does this mean?

Reply:
these two cases will be fixed in next pr. leave them here now to reminder to fix them.